src/styles/blocks/header.scss

```diff
@@ -0,0 +1,86 @@
+:root {
+  --navlink-active: #00acdc;
+}
+
+body {
+  box-sizing: border-box;
```

Comment:
this can be defined for * selector

Reply:
Not fixed 
<img width="840" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/1215da25-72c1-4527-bbdd-d88f595e0b2a">